src/components/conditional-wrapper/__tests__/ConditionalWrapper.test.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import ConditionalWrapper from '../ConditionalWrapper';
+
+describe('components/in-app-messenger/core/components/ConditionalWrapper', () => {
+    describe.each([[true], [false], [undefined]])('%o', isDisabled => {
+        test('renders correctly', () => {
+            const props = {
+                isDisabled,
+                wrapper: 'div',
+                otherProps: 'other-props',
+            };
+
+            const wrapper = shallow(
+                <ConditionalWrapper {...props}>
+                    <div />
+                </ConditionalWrapper>,
+            );
+
+            expect(wrapper).toMatchSnapshot();
```

Comment:
we're trying to avoid screenshots cause they're expensive. can we instead test for the correct components?